### PR TITLE
build the image everytime the machine is up (vagrant up)

### PR DESCRIPTION
**Pull request by i5o**

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -61,7 +61,8 @@ Vagrant.configure(2) do |config|
   #   vb.gui = true
   #
   #   # Customize the amount of memory on the VM:
-    vb.memory = "2222"
+    vb.memory = 4096
```
Comment (dogi) [inline Vagrantfile]: @i5o please leave default value ;)

File: Vagrantfile [hunk 2]
```diff
@@ -61,7 +61,8 @@ Vagrant.configure(2) do |config|
...
+    vb.cpus = 4
```
Comment (dogi) [inline Vagrantfile]: same here

File: Vagrantfile [hunk 3]
```diff
@@ -87,6 +88,9 @@ Vagrant.configure(2) do |config|
     #git checkout <branch>
     mkdir -p /vagrant/images
     ln -s /vagrant/images images
-    sudo -u vagrant screen -dmS build sudo bash -c 'export PATH="$PATH:/sbin:/usr/sbin";cd /home/vagrant/treehouse-builder;./treehouse-builder --chroot'
+  SHELL
+
+  config.vm.provision "shell",  run: "always", inline: <<-SHELL
+    sudo -u vagrant screen -dmS build sudo bash -c 'export PATH="$PATH:/sbin:/usr/sbin";cd /home/vagrant/treehouse-builder;./treehouse-builder --noninteractive; sudo poweroff'
```
Comment (dogi) [inline Vagrantfile]: better
`sudo -u vagrant screen -dmS build sudo bash -c 'export PATH="$PATH:/sbin:/usr/sbin";cd /home/vagrant/treehouse-builder;./treehouse-builder --noninteractive`

**Review by dogi — DISMISSED** [review]

**Comment by dogi** [comment]
hmm it got changed during my review ... hope all went fine there

File: Vagrantfile (hunk 3 above)
Comment (i5o) [reply Vagrantfile]: Do we want to keep it on once treehouse-builder finishes?
what for?

**Comment by i5o** [comment]
works better with https://github.com/ole-vi/treehouse-builder/pull/86

**Comment by dogi** [comment]
@i5o I did some changes
can you check if that makes sense?

**Review by dogi — APPROVED** [review]

**Pull request merged**